blog-publisher/generated/graphql.ts

```diff
@@ -0,0 +1,5002 @@
+import {TypedDocumentNode as DocumentNode} from "@graphql-typed-document-node/core"
```

Comment:
This file doesn't need to be in git. We can autogenerate it from the schema.graphql